desktop/cmp/dash/canvas/DashCanvasViewModel.ts

```diff
@@ -4,28 +4,31 @@
  *
  * Copyright © 2022 Extremely Heavy Industries Inc.
  */
-import {DashViewModel} from '@xh/hoist/desktop/cmp/dash/DashViewModel';
+import {DashViewConfig, DashViewModel} from '../DashViewModel';
 import '@xh/hoist/desktop/register';
 import {createObservableRef} from '@xh/hoist/utils/react';
-import {action, makeObservable, observable} from 'mobx';
+import {action, makeObservable, observable} from '@xh/hoist/mobx';
+import {ReactNode} from 'react';
 
 /**
  * Model for a content item within a DashCanvas.
- * @extends DashViewModel
  */
 export class DashCanvasViewModel extends DashViewModel {
-    /** @member {RefObject<DOMElement>} */
-    ref = createObservableRef();
-    /** @member {boolean} */
-    @observable hidePanelHeader;
-    /** @member {boolean} */
-    @observable hideMenuButton;
-    /** @member {Array} */
-    @observable.ref headerItems = [];
-    /** @member {boolean} */
-    @observable autoHeight;
+    ref = createObservableRef<HTMLElement>();
 
-    constructor(cfg) {
+    /** Hide the Header Panel for the view? Default false. */
+    @observable hidePanelHeader: boolean;
+
+    /** Hide the menu button for the view? Default false. */
+    @observable hideMenuButton: boolean;
+
+    /** Should the view resize its height to fit its contents? */
+    @observable autoHeight: boolean;
+
+    /** Additional items to include in header. */
+    @observable.ref headerItems: ReactNode[] = [];
```

Comment:
Do we want these doc comments on the class members?  They are not-quite-duplicates of the comments on the `DashCanvasViewSpec` interface properties.

Clearly some classes will have properties that aren't directly part of their ctor interface, but in the cases where they are, we might be best served by commenting them in one place, the interface app devs will use. Or if we want them in both places, it would be worth it to copy the same text.  Thoughts?